src/Rack.php

```diff
@@ -475,7 +477,11 @@ public static function showForRoom(DCRoom $room)
             echo "</tbody></table>";
         }
 
-        echo "<style>";
+        echo "<style>
+            :root {
+                --grid-room-cell-w: {$cell_w}px;
+                --grid-room-cell-h: {$cell_h}px;
```

Comment:
As it is a global variable, using `dcroom` as a prefix may be preferable. Also, I propose `dcroom` instead of `room` to match the itemtype classname.
```suggestion
                --dcroom-grid-cell-w: {$cell_w}px;
                --dcroom-grid-cell-h: {$cell_h}px;
```